code-of-conduct.md

```diff
@@ -54,7 +54,7 @@ further defined and clarified by project maintainers.
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
+Instances of abusive, harassing or otherwise unacceptable behavior may be
```

Comment:
:) 
You need to be consistent with your use of these commas, compared to your change in CONTRIBUTING.md